openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -112,19 +121,56 @@ class CourseFactory(DictFactoryBase):
     uuid = factory.Faker('uuid4')
 
 
+class JobOutlookItemFactory(DictFactoryBase):
+    value = factory.Faker('sentence')
+
+
+class PersonFactory(DictFactoryBase):
+    bio = factory.fuzzy.FuzzyText()
```

Comment:
You could probably use [ `Faker('paragraphs')` ](https://faker.readthedocs.io/en/master/providers/faker.providers.lorem.html)